lib/config/index.js

```diff
@@ -0,0 +1,20 @@
+// Make sure we process any environment variables first
+require('environment');
+
+// development, staging, testing or production
+const environment = process.env.NODE_ENV || 'development';
+
+const configMappings = {
+    dev: 'dev',
+    development: 'dev',
+    prod: 'prod',
+    production: 'prod',
+    staging: 'staging',
+    test: 'test',
+    testing: 'test'
+};
+
+const common = require('config/common');
+const envSpecific = require(`config/${configMappings[environment]}`);
+
+module.exports = {...common, ...envSpecific};
```

Comment:
GitHub is complaning about `no newline at end of file`, it should not make any difference, but let's remove that github red stop sign ;)

Reply:
Done :)